src/test/java/com/sweng894/GetVaccinated/VaccineTests.java

```diff
@@ -1,6 +1,6 @@
 package com.sweng894.GetVaccinated;
 
-import com.sweng894.GetVaccinated.model.Vaccine;
+import com.sweng894.GetVaccinated.api.entity.Vaccine;
```

Comment:
This is removing test coverage from from the pre-existing model.Vaccine class. We should have a new suite of tests for the new vaccine class or consolidate onto one Vaccine class.